### ISSUE TITLE
Manual update does not work anymore

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/haxe.org#118]
```
Failed to checkout branch master
Process creation failure : git
```

**Comment by Simn** [comment]
Note that I'm preparing to change the repository structure so the new markdown content for the website will be here: https://github.com/Simn/HaxeManual/tree/repository-restructure/output/HaxeManual/website

Where do we adjust this?

**Comment by Simn** [comment]
By "here" I mean output/HaxeManual/website, the repository is still going to be under HaxeFoundation/HaxeManual.

**Comment by jasononeil** [comment]
Update done.  

The initial error was from the memory leak, which is hopefully fixed with 944f90b990b678349f33a2517d7006f2ee00ee66

I've updated it to use the new markdown location as well. See f608a7035e35d02eab37c843715a06634de63eba for the change - basically it's a config value in `src/conf/app.json`
